### DISCUSSION TITLE
static media failure

### DISCUSSION THREAD

**Discussion by andrewm659** [netbox-community/netbox#7713] [Q&A]
I am getting the following error on my RHEL 8.4.1 server.  I was able to do the same thing on a AlmaLinux 8.4.1 server at home with SELinux on Enforcing mode.

I have tried running collectstatic and deleting the netbox folder and starting over and not running it.  It still fails for me at work.  Not sure why.  

```
Static Media Failure
The following static media file failed to load: netbox-print.css

Check the following:

    manage.py collectstatic was run during the most recent upgrade. This installs the most recent iteration of each static file into the static root path.
    The HTTP service (e.g. nginx or Apache) is configured to serve files from the STATIC_ROOT path. Refer to the installation documentation for further guidance.
        STATIC_URL: /static/
    The file netbox-print.css exists in the static root directory and is readable by the HTTP process.

Click here to attempt loading NetBox again.

```

**Comment by francescor** [comment]
same issue with us.

Everything works if we click on the "Click here to attempt loading NetBox again"

any suggestion?

**Comment by candlerb** [comment]
"I have tried running collectstatic and deleting the netbox folder" - which netbox folder did you delete exactly? And what exactly did you see when running manage.py collectstatic? Did you remember to activate the virtualenv first?

**Comment by Shea766** [comment]
Same thing for me except that when I click the try again link it still fails. I activated the virtualenv first. I'm done the collectstatic. I turned on the debug and the netbox-print.css file is there.

**Comment by robertlynch3** [comment]
I just went through this, I didn't realize you already posted about it. #8891

The resolution for me was `sudo setsebool -P httpd_can_network_connect 1`.

**Comment by JustRLC** [comment]
I think your main issue here is that you are not proxying to port 8001. Use Nginx and Apache and tell me what error you get. As it clearly states in the message. ` The HTTP service (e.g. nginx or Apache) is configured to serve files from the STATIC_ROOT path. Refer to the installation documentation for further guidance.
        STATIC_URL: /static/`
